sdk/lib/drivers/wdf/wdfdriverentry/wdfdriverentry.c

```diff
@@ -0,0 +1,87 @@
+/*
+ * PROJECT:     ReactOS WdfLdr driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Wdf driver intialization
+ * COPYRIGHT:   Copyright 2021 Max Korostil (mrmks04@yandex.ru)
+ */
+
+
+#include <ntddk.h>
+#include <windef.h>
+#include "wdf.h"
+#include "../wdfldr/wdfldr.h"
+
+
+const WDFFUNC *WdfFunctions;
+PWDF_DRIVER_GLOBALS WdfDriverGlobals;
+WDF_BIND_INFO BindInfo = {sizeof(WDF_BIND_INFO), L"KmdfLibrary", {1, 17, 7600}, WdfFunctionTableNumEntries, (void(__fastcall**)())(&WdfFunctions), NULL};
+PDRIVER_UNLOAD pOriginalUnload;
+DECLARE_UNICODE_STRING_SIZE(gRegistryPath, MAX_PATH);
+
+extern
+NTSTATUS
+NTAPI
+DriverEntry(
+    _In_ PDRIVER_OBJECT DriverObject,
+    _In_ PUNICODE_STRING RegistryPath);
+
+VOID
+NTAPI
+FxDriverUnloadCommon()
+{
+    WdfVersionUnbind(&gRegistryPath, &BindInfo, (PWDF_COMPONENT_GLOBALS)WdfDriverGlobals);
```

Comment:
Is there anything else here in the "real" `wdfdriverentry.lib`?

Reply:
Additionaly unbind classes. But now not implemented bind and unbind classes.